src/components/RuptureGenerationTask.tsx

```diff
@@ -20,6 +24,20 @@ const ruptureGenerationTaskQuery = graphql`
         created
         result
         state
+        files {
+          edges {
+            node {
+              role
+              file {
+                ... on File {
+                  id
+                  file_name
+                  file_url
```

Comment:
FYI @wooneh this is a slightly simpler and slightly costlier approach than the one I'd described, due to the AWS roundtrip time associated with the the file_url  resolver. But it works and get us going so we can do the 'cheaper' one later if we need.

Reply:
Noted, thanks!